dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/services/conversion/OdsToOdsConverter.kt

```diff
@@ -0,0 +1,54 @@
+package org.dataland.documentmanager.services.conversion
+
+import org.dataland.datalandbackendutils.exceptions.InvalidInputApiException
+import org.odftoolkit.simple.SpreadsheetDocument
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import org.springframework.web.multipart.MultipartFile
+import xyz.capybara.clamav.ClamavClient
+
+/**
+ * Validates the content of an ods file
+ */
+@Component
+class OdsToOdsConverter(
```

Comment:
if its just a validator, why does it have the name converter?

Reply:
I'm also unhappy with the name. These classes do two things validation and conversion. I could not come with a better name unifying these. `Processor` felt too general. And splitting these classes up would mess with the detection logic. What is your opinion?

Reply:
okay that's a valid reason to leave it that way imo